xla/myplugin/myplugin_cpp_pjrt.cc

```diff
@@ -0,0 +1,13 @@
+#include "myplugin_cpp_pjrt.h"
+
+namespace myplugin_pjrt {
+
+absl::string_view MypluginPjrtClient::platform_name() const {
```

Comment:
Any reason why these are in a CC and everything else inlined in the header?

Reply:
No particular reason, more like everything in PJRT_Client.h already had these in the header. So this was more of an example of using the CC. Any strong preference to move these to CC?

I also wanted to update the default implementation to return error unimplmented so theres less boilerplate for everyone else.